config/site/src/_includes/navbar.html

```diff
@@ -65,11 +69,11 @@
           </span>
           <span>Toggle Theme</span>
         </button>
-        <a href="{{ site.github_url }}" target="_blank" class="text-base {{ site.style.link }} inline-flex items-center space-x-2" rel="noopener">
+        <a href="{{ site.github_url }}" target="_blank" class="text-base {{ site.style.navbar_link }} inline-flex items-center space-x-2" rel="noopener">
           {% include icons/github.svg %}
```

Comment:
Feels like this icon could be larger:

![image](https://github.com/user-attachments/assets/9318b8f2-6aee-4874-a0ed-fa05b3c74ffe)

And the dark/light toggle as well? They could be the height of the search box. Thoughts?